microsoft-edge/webview2/get-started/hololens2.md

```diff
@@ -0,0 +1,594 @@
+
+To view the `WebView` script you can double-click on it in the Inspector window. This should open it in your default code editor.
+
+Now that you've reviewed the `WebView` code, let's extend the functionality of our sample. To do that, we'll create a new script that derives from `WebView`.
+
+1. In the Inspector window for WebView, click **Add Component**, select **New script**, and type `WebViewBrowser`, click **Create and add**. You new component will be added to the Inspector window.
+
+1. Double-click on the `WebViewBrowser` script in the Inspect window to edit the script.
+
+1. Replace the contents of that file with the code below.
+
+```C#
+using Microsoft.MixedReality.WebView;
+using UnityEngine.UI;
+using UnityEngine;
+using TMPro;
+using System;
+
+public class WebViewBrowser: MonoBehaviour
+{
+    // Declare UI elements: back button, go button, and URL input field
+    public Button BackButton;
+    public Button GoButton;
+    public TMP_InputField URLField;
+
+    private void Start()
+    {
+        // Get the WebView component attached to the game object
+        var webViewComponent = gameObject.GetComponent<WebView>();
+        webViewComponent.GetWebViewWhenReady((IWebView webView) =>
+        {
+            // If the WebView supports browser history, enable the back button
+            if (webView is IWithBrowserHistory history)
+            {
+                // Add an event listener for the back button to navigate back in history
+                BackButton.onClick.AddListener(() => history.GoBack());
+
+                // Update the back button's enabled state based on whether there's any history to go back to
+                history.CanGoBackUpdated += CanGoBack;
+            }
+
+            // Add an event listener for the go button to load the URL entered in the input field
+            GoButton.onClick.AddListener(() => webView.Load(new Uri(URLField.text)));
+
+            // Subscribe to the Navigated event to update the URL input field whenever a navigation occurs
+            webView.Navigated += OnNavigated;
+
+            // Set the initial value of the URL input field to the current URL of the WebView
+            if (webView.Page != null)
+            {
+                URLField.text = webView.Page.AbsoluteUri;
+            }
+        });
+    }
+
+    // Update the URL input field with the new path after navigation
+    private void OnNavigated(string path)
+    {
+        URLField.text = path;
+    }
+
+    // Enable or disable the back button based on whether there's any history to go back to
+    private void CanGoBack(bool value)
+    {
+        BackButton.enabled = value;
+    }
+}
+```
+
+1. Modify the scene hierarchy as follows:
```

Comment:
@mikehoffms any idea what keeps happening to the numbering? It looked correct on my local machine.

Reply:
@michaelfarnsworth - i'm in the file, will fix by shifting the above code block right to align w/ text to right of "1. "

Reply:
i have the Learn Authoring Pack installed.  after shift right, VS Code's preview (w/ pak) looks ok #ing

Reply:
fixed, see # below code listing in 
https://github.com/michaelfarnsworth/edge-developer/blob/webview2-hololens-getting-started/microsoft-edge/webview2/get-started/hololens2.md#step-6---configure-webview2-settings-and-properties

you can Resolve this conversation